pages/agent/v3/help/_lock_release.md

```diff
@@ -20,25 +20,30 @@ script.
 Releases the lock for the given key. This should only be called by the
 process that acquired the lock. To help prevent different processes unlocking
 each other unintentionally, the output from `lock acquire` is required as the
-second argument (`token`).
+second argument, i.e. `token`.
```

Comment:
Any particular reason for this change? We normally try to avoid "i.e." and "e.g." because people confuse them. It's better to right the full english, or maybe keep the original version here 🤔

Reply:
I read the version with the parens and stumped. I do think `i.e.` is clearer than symbols *if* people understand what it expands to. So I will expand it.

Reply:
```suggestion
second argument, namely, the `token` in the Usage section above.
```